client/js/models/tag_category.js

```diff
@@ -58,16 +68,19 @@ class TagCategory extends events.EventTarget {
         if (this.color !== this._origColor) {
             detail.color = this.color;
         }
+        if (this.order !== this._origOrder) {
+            detail.order = this.order;
+        }
 
         if (!Object.keys(detail).length) {
             return Promise.resolve();
         }
 
         let promise = this._origName
             ? api.put(
-                  uri.formatApiLink("tag-category", this._origName),
-                  detail
-              )
+                uri.formatApiLink("tag-category", this._origName),
+                detail
+            )
```

Comment:
Should install and enable [pre-commit](https://pre-commit.com/) to ensure everything is of a consistent style

Reply:
I believe this is the correct formatting, and that it was wrong before. pre-commit also formats it like this.